### DISCUSSION TITLE
Poner los valores en cero

### DISCUSSION THREAD

**Discussion by DeveloperJF** [iSoron/uhabits#1764] [Feature Requests]
Muy buen día;

He estado utilizando esta App, me parece muy buena aunque hay algunas caracterísiticas que podrían añadir y otras en mejorar. En este caso estoy viendo que la App no tiene la opción de restaurar todos los hábitos de sus valores (rachas, frecuencia, calendario, historial, puntuación, resumen) o uno por uno de los hábitos que uno haya creado y que quiere volver a reiniciar, con el fin de conservar las características con la que se creó inicialmente el hábito y volviéndo toda la información desde cero (Estadística y demás). Porque en este caso por lo que veo actualmente solo existe la opción de eliminar este hábito y volver a configurarlo para que vuelva a empezar desde cero, es lo único que  veo que se puede hacer, ¿O si existe y no me he dado cuenta?; espero sus comentarios y ayuda sobre el tema.

Para darme a entender mejor les comento: Hace mucho que no volvía a utlizar la App por ende (rachas, frecuencia, calendario, historial, puntuación, resumen) que tengo en cada uno de los hábitos lo quiero reiniciar para que empiece a hacer el registro desde el día de hoy y no con la fecha en que lo creé el hábito, en mi caso desde el 2017. Es a esto a lo que me refiero, me gustaría que hubiera una opción para REINICIAR estos datos y quedar desde el día de hoy sin tener que eliminar el hábito.

Espero haberme dado a entender con el ejemplo y con el archivo adjunto que dejé acá.

Cordialmente;

JUAN JIMÉNEZ
![photo_2023-07-19_10-10-53](https://github.com/iSoron/uhabits/assets/37987294/02286b3c-12e8-46a6-bdc2-80f843679d23)

**Comment by exanrauzer** [comment]
¡Hola! Por favor, ¿podrías responder por qué estás usando tu idioma español en un servicio que no tiene funcionalidad de traducción automática?

**Reply by marylicious** [reply]
![image](https://github.com/iSoron/uhabits/assets/65676201/b30aedde-9dc7-4e47-acb2-7c8eebada762)
Entiendo tu punto pero Github si traduce
